ext/standard/var.c

```diff
@@ -1241,15 +1247,15 @@ static void php_var_serialize_intern(smart_str *buf, zval *struc, php_serialize_
 				if (count > 0 && incomplete_class) {
 					--count;
 				}
-				php_var_serialize_nested_data(buf, struc, myht, count, incomplete_class, var_hash);
+				php_var_serialize_nested_data(buf, struc, myht, count, incomplete_class, var_hash, GC_REFCOUNT(myht) == 1);
```

Comment:
I'm not sure if we need ``GC_REFCOUNT(myht) == 1`` here. Shouldn't it be just ``true``?

Reply:
I was wrong. Your code is right.